specs/core/0_beacon-chain.md

```diff
@@ -966,7 +966,8 @@ def get_domain(state: BeaconState,
     """
     epoch = get_current_epoch(state) if message_epoch is None else message_epoch
     fork_version = state.fork.previous_version if epoch < state.fork.epoch else state.fork.current_version
-    return bytes_to_int(fork_version + int_to_bytes4(domain_type))
+    # fork version is on the big-endian side: when signing using only the type (e.g. deposits), the type can be passed directly.
```

Comment:
Design rationales will go in another document. Suggest removing this comment. (As a side note, comments start with a capital.)

Reply:
Should have been `fork_version`, hence no capital. I'll remove it.